@ui/ui-kit/Icon/IconsComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import { MakeIcon } from './MakeIcon'
+import { IconsPropsType } from './iconsType'
+import HomeOutline from '@/public/sidebar-icons/home-outline.svg'
+import Home from '@/public/sidebar-icons/home.svg'
+import CreateOutline from '@/public/sidebar-icons/plus-square-outline.svg'
+import Create from '@/public/sidebar-icons/plus-square.svg'
+import ProfileOutline from '@/public/sidebar-icons/person-outline.svg'
+import Profile from '@/public/sidebar-icons/person.svg'
+import MessageOutline from '@/public/sidebar-icons/message-circle-outline.svg'
+import Message from '@/public/sidebar-icons/message-circle.svg'
+import SearchOutline from '@/public/sidebar-icons/search-outline.svg'
+import Search from '@/public/sidebar-icons/search.svg'
+import TrendingUpOutline from '@/public/sidebar-icons/trending-up-outline.svg'
+import TrendingUp from '@/public/sidebar-icons/trending-up.svg'
+import BookmarkOutline from '@/public/sidebar-icons/bookmark-outline.svg'
+import Bookmark from '@/public/sidebar-icons/bookmark.svg'
+import LogoutOutline from '@/public/sidebar-icons/log-out-outline.svg'
+import Logout from '@/public/sidebar-icons/log-out.svg'
```

Comment:
it's better to create an index file in the folder where all the svgs are located, and access them from here with the file index

Reply:
done